### DISCUSSION TITLE
Welcome to melfa_ros2_driver Discussions!

### DISCUSSION THREAD

**Discussion by offinliu** [Mitsubishi-Electric-Asia/melfa_ros2_driver#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
 * Feel free to ask any questions you have about MELFA ROS2 Driver.
### Brief Introduction 
MELFA ROS2 Driver is co-developed between ROS Industrial Asia Pacific and Mitsubishi Electric Asia.
This driver is OEM developed, OEM optimised and falls under after-sales OEM support worldwide.

**Comment by rkent** [comment]
I'm curious why you have not added this to rosdistro, so that it gets displayed in official ROS locations. (I'm the guy who maintains rosindex, used for package discovery, and rosdoc2, that shows individual package documentation.) If you don't want ROS to host the binaries, you can still add a 'doc' entry so that the package is indexed and discoverable.

This is not a complaint, I'm genuinely curious. If you'd rather reply privately, I'm available at kent@caspia.com

**Reply by offinliu** [reply]
Hi Kent, 
I've sent you an email.
